src/components/Dashboard.vue

```diff
@@ -82,49 +97,55 @@ export default {
     },
     stopEvent () {
       event.stopPropagation()
+    },
+    openSendModal (id) {
+      // モーダルウィンドウ表示
+      this.showSendModal = true
+      this.destinationId = id
+    },
+    closeSendModal () {
+      this.showSendModal = false
+    },
+    updateWallet () {
```

Comment:
ここにasyncをつけて、store.dispatchの前にawaitをつけるといかがでしょうか？
```suggestion
    async updateWallet () {
```

もしくはdispathがPromiseを返却するのであれば、async / awaitでなくthenで繋ぐことも可能です。
`store.dispatch(,,,,,).then()`

が、現場ではほぼほぼasync / awaitを使用するので、今から慣れておくのも良いかと思います。